docker-compose.yml

```diff
@@ -88,14 +88,12 @@ services:
       - "logging=true"
 
   # SPARQL Endpoint authorization service. Adds an authentication layer to the triplestore
-  # https://github.com/mu-semtech/mu-authorization
+  # https://github.com/mu-semtech/sparql-parser
   database:
-    image: semtech/mu-authorization:0.6.0-beta.8
-    environment:
-      MU_SPARQL_ENDPOINT: "http://triplestore:8890/sparql"
-      ERROR_ON_UNWRITTEN_DATA: "on"
```

Comment:
@madnificent Is there a equivalent in the new mu-auth?

Reply:
TY for the message on the chat.  It's turned on by default and the environment variable can be removed (as well `MU_SPARQL_ENDPOINT`).